CHARTER.md

```diff
@@ -0,0 +1,192 @@
+election. Elections shall be done within the projects by the Collaborators active in the project.
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for 
+TSC meetings and a OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a 
+term of one year. The Chair and voting CPC member may be (but are not required to be) the same person. The TSC shall 
+hold annual elections to select a TSC Chairperson and voting CPC member; there are no limits on the number of terms a 
+TSC Chairperson or voting CPC member may serve.
+
+#### Section 4.2.2. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish 
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may 
+choose to participate in any vote through abstention.
+
+All changes to this charter must be approved by the CPC.
+
+#### Section 4.2.3. Project Roles
+
+The webpack git repository is maintained by the TSC and additional Collaborators who are added by the TSC voting members 
+on an ongoing basis.
+
+Individuals making significant and valuable contributions are made Collaborators and given commit-access to the project.
+These individuals are identified by the TSC and their addition as Collaborators is discussed during a TSC meeting. 
+Modifications of the contents of the git repository are made on a collaborative basis as defined in the development 
+process.
+
+Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
+consensus to the TSC for discussion by assigning the `tsc-agenda` tag to a pull request or issue. The TSC voting members 
+should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
+well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base. 
+  Contributors work with the project's Collaborators to have code committed to the code base. A Contributor may be 
+  promoted to a Collaborator by the TSC. Contributors should rarely be encumbered by the TSC and never by the CPC or 
+  OpenJS Foundation Board.
+
+* **Project**: a technical collaboration effort, e.g. a subsystem, that is organized through the project creation 
```

Comment:
Use `webpack project` or `internal project` here to distinguish it from what the foundation calls projects.

Reply:
fixed.